pkg/lifecycle/lifecycle.go

```diff
@@ -114,25 +118,28 @@ type Transition struct {
 	XMLName      xml.Name       `xml:"Transition" json:"-"`
 	Date         ExpirationDate `xml:"Date,omitempty" json:"Date,omitempty"`
 	StorageClass string         `xml:"StorageClass,omitempty" json:"StorageClass,omitempty"`
-	Days         ExpirationDays `xml:"Days,omitempty" json:"Days,omitempty"`
+	Days         ExpirationDays `xml:"Days" json:"Days"`
```

Comment:
StorageClass is not optional - so should not be `omitempty`. Otherwise the IsNull checks while marshaling make no sense